packages/web-components/fast-foundation/src/listbox-option/listbox-option.ts

```diff
@@ -102,14 +107,14 @@ export class ListboxOption extends FoundationElement {
     @observable
     public selected: boolean = this.defaultSelected;
     protected selectedChanged(): void {
-        if (this.$fastController.isConnected) {
-            if (!this.dirtySelected) {
-                this.dirtySelected = true;
-            }
+        this.ariaSelected = this.selected ? "true" : "false";
 
-            if (this.proxy instanceof HTMLOptionElement) {
-                this.proxy.selected = this.selected;
-            }
+        if (!this.dirtySelected) {
+            this.dirtySelected = true;
+        }
```

Comment:
would you mind adding. comment on this change just for future reference?

Reply:
The `selected` property doesn't rely on the component being connected, so removing the `isConnected` check here allows the `aria-selected` attribute to be set before connecting as well.